cmd/resources/resources.go

```diff
@@ -210,7 +210,7 @@ func NewResourceCmd(
 		},
 	}
 
-	cmd.SetUsageTemplate(getResourceUsageTemplate())
+	cmd.SetUsageTemplate(OperationUsageTemplate())
```

Comment:
Is this still an accurate name, or should we rename it to not be resource- or operation-specific?

Reply:
yeah maybe `SubcommandUsageTemplate`?